### ISSUE TITLE
Feature request/idea: Add Assign to Overlay Maintainer

### ISSUE THREAD

**Issue by jonasstein** [mgorny/bug-assign-user-js#3]
It takes a lot of time to lookup the maintainer of an overlay.
Would it be possible to read 
https://api.gentoo.org/overlays/repositories.xml
and create an drop down list similar to the magenta sketch in the attachment?
![selection_283](https://cloud.githubusercontent.com/assets/298682/26278519/8339330c-3d9c-11e7-8bad-a332767b2861.png)

**Comment by mgorny** [comment]
I don't think that would be that friendly, it would probably be better if we started using `::repo` syntax, and then handled non-gentoo repos directly. However, I'm not really into web-stuffs, so I'd prefer if someone else did it ;-).
